spec/models/page_spec.rb

```diff
@@ -421,14 +429,14 @@
         expect(child.path).to eq('/parent/child')
       end
     end
-  end
+  end # path
```

Comment:
not necessary but it helps to navigate in files with long blocks to know where the `end` belongs to.